data/typelibrary/adapterized-1.0.0/typelib/io/IXP.fbt

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="IXP" Comment="Composite FB for boolean input data" >
```

Comment:
why P and not A or ADP?

Reply:
So IXA and QXA ist better. 
I fully agree.


As Plug and Socket are sub-ideas of Adapters. 
Also it would be IXP and QXS which is hard to keep in mind.